src/components/protocol_handler/test/incoming_data_handler_test.cc

```diff
@@ -56,6 +56,8 @@ class IncomingDataHandlerTest : public ::testing::Test {
     some_data2_size = 512;
     some_data = new uint8_t[some_data_size];
     some_data2 = new uint8_t[some_data2_size];
+    memset(some_data, 0, some_data_size);
```

Comment:
@AGaliuzov Yes, it fixes reported by @pestOO Valgrind problems of using uninitialized values.